### PR TITLE
hanlder_click_item

### PR DESCRIPTION
**Pull request by tungct97**
#8 refs_5187_Handle event click navigation screen on RecyclerView
#Redmine https://edu-redmine.framgia.vn/issues/5187

### CODE REVIEW

**Comment by tungct97** [comment]
ready

File: app/src/main/java/com/framgia/music_51/screen/detail_genre/DetailGenreActivity.java [hunk 1]
```diff
@@ -44,18 +48,35 @@ protected void onCreate(Bundle savedInstanceState) {
...
         mBinding = DataBindingUtil.setContentView(this, R.layout.activity_detal_genre);
         mViewModel = ViewModelProviders.of(DetailGenreActivity.this).get(DetailGenreViewModel.class);
-        mAdapter = new DetailAdapter();
+        mAdapter = new DetailAdapter(this);
         getGenre(getMusicGenreIntent());
         getData();
+        mLoadMore = new OnLoadMore(new LinearLayoutManager(this)) {
+            @Override
+            public void onLoadMore() {
+                new Handler().postDelayed(new Runnable() {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/music_51/screen/detail_genre/DetailGenreActivity.java]: tại sao lại cần delay?

File: app/src/main/java/com/framgia/music_51/screen/detail_genre/OnLoadMore.java [hunk 2]
```diff
@@ -0,0 +1,31 @@
+package com.framgia.music_51.screen.detail_genre;
+
+import android.support.annotation.NonNull;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+
+public abstract class OnLoadMore extends RecyclerView.OnScrollListener {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/music_51/screen/detail_genre/OnLoadMore.java]:
```suggestion
public abstract class EnlessScrollListener extends RecyclerView.OnScrollListener {
```

File: app/src/main/java/com/framgia/music_51/screen/play/PlayActivity.java [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+import android.content.Intent;
+import android.databinding.DataBindingUtil;
+import android.os.Bundle;
+import android.support.v7.app.AppCompatActivity;
+
+import com.framgia.music_51.R;
+import com.framgia.music_51.data.model.Track;
+import com.framgia.music_51.databinding.ActivityPlayBinding;
+
+public class PlayActivity extends AppCompatActivity {
```
Comment (DoanVanToan) [inline app/src/main/java/com/framgia/music_51/screen/play/PlayActivity.java]:
```suggestion
public class PlayerActivity extends AppCompatActivity {
```

**Comment by tungct97** [comment]
ready

**Review by DoanVanToan — APPROVED** [review]

**Pull request merged**